### PR TITLE
sign up user

### PR DESCRIPTION
**Pull request by nhatnguyen71298**
https://edu-redmine.sun-asterisk.vn/issues/34848

- Chức năng sign up.
- 
Screenshot from 2021-03-24 17-05-25

Screenshot from 2021-03-24 17-06-33

### CODE REVIEW

**Comment by nhatnguyen71298** [comment]
ready

File: app/views/shared/_error_messages.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<% if @user.errors.any? %>
+  <div id="error_explanation">
+    <div class="alert alert-danger">
+      <%= t("error", count: object.errors.count) %>.
+    </div>
+    <ul>
+      <% @user.errors.full_messages.each do |msg| %>
```
Comment (vanvtt-0952) [inline app/views/shared/_error_messages.html.erb]: sao lại user nhỉ

Comment (nhatnguyen71298) [reply app/views/shared/_error_messages.html.erb]: tào lao đó chị :v

**Comment by nhatnguyen71298** [comment]
ready

**Pull request merged**